platform/ui/src/components/Button/Button.tsx

```diff
@@ -38,41 +39,92 @@ const variantClasses = {
   },
   outlined: {
     default:
-      'border bg-trasparent border-primary-light text-primary-light hover:bg-primary-light hover:text-black focus:text-black focus:bg-primary-light active:opacity-80',
+      'text-primary-light hover:bg-primary-light hover:text-black focus:text-black focus:bg-primary-light active:opacity-80',
```

Comment:
Not your fault but just sharing my frustration  when I see things like this on OHIF. Not DRY at all